### PR TITLE
add blog post for treetables plugin

### PR DESCRIPTION
**Pull request by hillalex**

### CODE REVIEW

File: content/blog/treetables-1.1.1.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+title: jQuery TreeTables 1.1.1
+best: false
+---
+
+We just released a new npm package [TreeTables](https://www.npmjs.com/package/treetables)
+ for displaying tree data when using jQuery
+[DataTables](https://datatables.net/)
+
+DataTables is a powerful and extremely well established plugin for
+displaying tabular data, but has no native support for tree data.
```
Comment (richfitz) [inline content/blog/treetables-1.1.1.md]: can you give an example of tree data?

**Review by richfitz — COMMENTED** [review]
One suggestion and also a gif might be nice? 
Peek 2019-06-12 14-27

File: content/blog/treetables-1.1.1.md (hunk 1 above)
Comment (EmmaLRussell) [reply content/blog/treetables-1.1.1.md]: Maybe 'nested data' would be more obvious than 'tree data'?

**Review by EmmaLRussell — COMMENTED** [review]
Looks good to me, though I think you could big up its cool features a little more, like arbitrary levels of nesting, expand/collapse all, persistence of collapse state etc. (Obvious from the image but might be something someone's searching for specifically..)

**Comment by hillalex** [comment]
Thanks for the suggestions both! Added the gif and a bit more explanation :) Rich how do you make those gifs?!

**Comment by richfitz** [comment]
For gifs I use https://github.com/phw/peek - you can fiddle with framerate and downsampling to get a reasonable size/quality ratio (this was 10fps)

**Review by richfitz — APPROVED** [review]
LGTM (you can drop that png but it's not a big deal)

**Pull request merged**